src/CompactErrorFormatter.php

```diff
@@ -15,13 +15,18 @@ class CompactErrorFormatter implements ErrorFormatter
 	/** @var string */
 	private $format;
 
+	/** @var string */
+	private $editorUrl;
+
 	public function __construct(
 		RelativePathHelper $relativePathHelper,
-		string $format
+		string $format,
+		?string $editorUrl
 	)
 	{
 		$this->relativePathHelper = $relativePathHelper;
 		$this->format = $format;
+		$this->editorUrl = $editorUrl ?? '';
```

Comment:
What happens if this falls back to empty string? I presume symfony prints the escape sequence with an empty href – are terminals ok with it?

Reply:
I would assume, that `null` variant cannot happen, therefore it should be removed. Because there should be a reasonable default. In case it is missing, maybe write a notice into stderr output?

Reply:
https://github.com/phpstan/phpstan-src/blob/fed28bc6a9d413c373dda0a8fc598c7ff8035977/conf/config.neon#L280

Reply:
I think this is fine as it is. If you want to use the `editorUrl` then you already know that you need to set it (as per readme). Also, if you are using a custom error formatter, you kind of know what you are doing and also test it.

Reply:
Please add the error message to phpstan output & reasonable default when `editorUrl` is null.